fiftyone/core/stages.py

```diff
@@ -5319,6 +5430,7 @@ def _kwargs(self):
         return [
             ["field_names", self._field_names],
             ["_allow_missing", self._allow_missing],
+            ["_meta_filter", self._meta_filter],
```

Comment:
In addition to adding new parameters to `_kwargs()`, they also need to be added to `_params()`. The latter declares the type(s) of the arguments so that the App can properly render instances of the stage in the view bar.